apps/server/src/schema.graphql

```diff
@@ -39,25 +34,23 @@ input UserInput {
   companyName: String
 }
 
+ 
+
 input ReportInput {
   _id: ReportIdInput
   summary: String
   status: String
   reviews: ReviewsInput
 }
 
-input PeerUpdateInput {
-  newReviewerId: String!
-}
-
 input ReviewsInput {
   self: ReviewInput
-  peers: [ReviewInput]
+  peers: ReviewInput
```

Comment:
why isn't this an array? because you're never going to update 2+ peers at the same time?

that's fine, but if it's not an array, it shouldn't be 'peers' plural